prow/bump.sh

```diff
@@ -65,31 +65,40 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
   echo "Detected GOOGLE_APPLICATION_CREDENTIALS, activating..." >&2
   gcloud auth activate-service-account --key-file="$GOOGLE_APPLICATION_CREDENTIALS"
   gcloud auth configure-docker
-  cmd="--push"
 elif [[ $# != 0 ]]; then
```

Comment:
this should probably be changed to `if`, as otherwise `--push` will never work when `GOOGLE_APPLICATION_CREDENTIALS` is set.

though also: why is `--push` even a part of this script?

Reply:
Done

> why is --push even a part of this script?

That's originally all this script did. Might make sense soon to refactor it into a `push.sh` script soon